.github/workflows/pytest.yaml

```diff
@@ -28,17 +28,9 @@ jobs:
       - name: Upgrade pip
         run: python -m pip install --upgrade pip
 
-      - name: Install dev version of htmltools
+      - name: Install dev version of rstudio packages
         run: |
-          pip install wheel https://github.com/rstudio/py-htmltools/tarball/main
-
-      # - name: Check out and install fontawesome
-      #   run: |
-      #     cd ..
-      #     git clone https://github.com/rstudio/py-fontawesome.git
-      #     cd py-fontawesome
-      #     pip install -r requirements-dev.txt
-      #     make install
+          pip install wheel -r .github/requirements_gha.txt
```

Comment:
I think that using hyphen in the name `requirements-gha.txt` is more in line with naming conventions (like `requirements-dev.txt`).

Reply:
Renamed to `requirements-tox.txt`